components/studio/libexec/hab-studio-type-default.sh

```diff
@@ -119,6 +119,7 @@ sup-run() {
   echo "--> Launching the Habitat Supervisor in the background..."
   echo "    Running: hab sup run \$*"
   hab sup run \$* > /hab/sup/default/sup.log &
+  echo \$! > /hab/sup/default/launch.pid
```

Comment:
Haha, I was worried until I remembered what Bash gave us. This ends up being the critical bit so the behavior didn't need to change much.